recipes/simbody/all/test_package/conanfile.py

```diff
@@ -0,0 +1,24 @@
+from conan import ConanFile
+from conan.tools.build import cross_building
+from conan.tools.cmake import CMake, cmake_layout
+import os
+
+
+class TestPackageConan(ConanFile):
+    settings = "os", "compiler", "build_type", "arch"
+    generators = "cmake", "cmake_find_package"
+
+    test_type = "explicit"
+    def requirements(self):
+         self.requires(self.tested_reference_str)
+
+    def build(self):
+        cmake = CMake(self)
+        #cmake.definitions["SIMBODY_VER"] = tools.Version(self.deps_cpp_info["simbody"].version).major
+        cmake.configure()
+        cmake.build()
+
+    def test(self):
+        if not cross_building(self.settings):
```

Comment:
```suggestion
        if can_run(self.settings):
```
can_run it's a new flavor for Conan 2.x. More info: https://docs.conan.io/en/latest/reference/conanfile/tools/build.html?highlight=can_run#conan-tools-build-can-run